target/linux/ath79/dts/qca9557_dongwon_dw02-412h-64m.dts

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "qca9557_dongwon_dw02-412h.dtsi"
+
+/ {
+	model = "Dongwon T&I DW02-412H (64M)";
+	compatible = "dongwon,dw02-412h-64m", "qca,qca9557";
+};
+
+&nand {
+	status = "okay";
+
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		partition@0 {
+			label = "current";
+			reg = <0x0 0x1000000>;
+			read-only;
+		};
+
+		partition@1000000 {
+			label = "firmware";
+			reg = <0x1000000 0x3000000>;
+
+			compatible = "fixed-partitions";
+			#address-cells = <1>;
+			#size-cells = <1>;
+
+			partition@0 {
+				label = "kernel";
+				reg = <0x0 0x1000000>;
```

Comment:
I don't think we need that much space for kernel... How about shrinking this to 8MB? (though that will make the numbers slightly dirtier)

Reply:
I also thought that space was too large for kernel. I'll reduce it to 8MB.